shared-services/src/files/CloudinaryUpload.ts

```diff
@@ -0,0 +1,50 @@
+import axios from "axios"
+
+interface CloudinaryUploadProps {
+  file: File
+  onUploadProgress: (progress: number) => void
+  cloudName: string
+  uploadPreset: string
+  tag?: string
+  signature?: string
+  apiKey?: string
+  timestamp?: number
+}
+
+export const CloudinaryUpload = async ({
+  file,
+  onUploadProgress,
+  cloudName,
+  uploadPreset,
+  signature,
+  apiKey,
+  timestamp,
+  tag = "browser_upload",
+}: CloudinaryUploadProps) => {
+  const url = `https://api.cloudinary.com/v1_1/${cloudName}/upload`
+  const data = new FormData()
+  data.append("upload_preset", uploadPreset)
+  data.append("tags", tag)
+  data.append("file", file)
+  if (signature && timestamp && apiKey) {
+    data.append("signature", signature)
+    data.append("timestamp", `${timestamp}`)
+    data.append("api_key", apiKey)
+  }
+
+  if (!cloudName || cloudName == "" || !uploadPreset || uploadPreset == "") {
+    const err = "Please supply a cloud name and upload preset for Cloudinary"
+    alert(err)
```

Comment:
This is all existing code, I just moved it here since it was removed from ui-components in https://github.com/bloom-housing/bloom/pull/3393. I would rather leave it as is